### PR TITLE
Add UIKIT_EXPORT_CLASS macro missing to UIKit include files

### LINKED ISSUE #1053: UIKIT_EXPORT_CLASS macro missing for some UI classes
The UIKIT_EXPORT_CLASS macro is missing for some UI classes like UIMenuController and UIMenuItem.

### PR DESCRIPTION
**Pull request by tadam-msft**
Added missing UIKIT_EXPORT_CLASS macro for two classes.

### CODE REVIEW

File: include/UIKit/UIMenuController.h [hunk 1]
```diff
@@ -40,6 +40,8 @@ extern NSString* const UIMenuControllerMenuFrameDidChangeNotification;
 
 @class UIView, UIWindow;
 
+UIKIT_EXPORT_CLASS
```
Comment (yiyang-msft) [inline include/UIKit/UIMenuController.h]: in comments of original issue, UIRefreshControl has been called out to miss this as well. Mind adding UIKIT_EXPORT_CLASS  too to UIRefreshControl?

**Comment by yiyang-msft** [comment]
:shipit:

**Comment by DHowett-MSFT** [comment]
nit: can you reword the title to be <=72 chars and in imperative mood?
thanks!

**Comment by jaredhms** [comment]
:shipit:

**Pull request merged**